tensilelite/Tensile/LibraryIO.py

```diff
@@ -30,6 +30,22 @@
 from . import SolutionLibrary
 
 from typing import NamedTuple, List
+import os
+import sys
+
+try:
+    import orjson as json
+except ImportError:
+    try:
+        import ujson as json
+        printWarning("orjson not installed. Fallback to ujson.")
+    except ImportError:
+        try:
+            import simplejson as json
+            printWarning("orjson, ujson not installed. Fallback to simplejson.")
+        except ImportError:
+            import json
```

Comment:
Why not just using "json" but considering all variants? Apart from encoding, is there any further concern?

Reply:
There are performance gaps between these packages.

Reply:
OK got it.